tables/subject_vaccines/arora23-vacc.csv

```diff
@@ -0,0 +1,15 @@
+﻿ref_name,subject_name,vaccination_date_cmp,vaccination_date,vaccine_name,dosage,location,section
+Arora23,BNT162b2_3dose,~,2021-01-01,BNT162b2,1,Germany,NULL
+Arora23,BNT162b2_3dose,~,2021-02-01,BNT162b2,2,Germany,NULL
+Arora23,BNT162b2_3dose,~,2021-08-01,BNT162b2,3,Germany,NULL
+Arora23,vaccination_3dose_BA5_bti,~,2021-01-01,BNT162b2 or AZD1222,1,Germany,NULL
+Arora23,vaccination_3dose_BA5_bti,~,2021-02-01,BNT162b2 or AZD1222,2,Germany,NULL
+Arora23,vaccination_3dose_BA5_bti,~,2021-08-01,BNT162b2 or AZD1222,3,Germany,NULL
+Arora23,vaccination_3dose_monovalent_1dose,~,2021-01-01,BNT162b2 or AZD1222,1,Germany,NULL
+Arora23,vaccination_3dose_monovalent_1dose,~,2021-02-01,BNT162b2 or AZD1222,2,Germany,NULL
+Arora23,vaccination_3dose_monovalent_1dose,~,2021-08-01,BNT162b2 or AZD1222,3,Germany,NULL
+Arora23,vaccination_3dose_monovalent_1dose,~,2022-01-01,mRNA-monovalent,4,Germany,NULL
+Arora23,vaccination_3dose_bivalent_1dose,~,2021-01-01,Unknown vaccine,1,Germany,NULL
+Arora23,vaccination_3dose_bivalent_1dose,~,2021-02-01,Unknown vaccine,2,Germany,NULL
+Arora23,vaccination_3dose_bivalent_1dose,~,2021-08-01,Unknown vaccine,3,Germany,NULL
+Arora23,vaccination_3dose_bivalent_1dose,~,2022-01-01,mRNA-bivalent,4,Germany,NULL

```

Comment:
@zachandfox dose the paper provided the time between 3rd dose and 4th dose? otherwise use 1 year.

Reply:
Time is not provided, I assumed bivalent vaccines became available on 2022-01-01. I will change it to 1 year difference